bokehjs/src/lib/core/dom_view.ts

```diff
@@ -76,40 +76,55 @@ export abstract class DOMComponentView extends DOMElementView {
     this.shadow_el = this.el.attachShadow({mode: "open"})
   }
 
-  override styles(): StyleSheetLike[] {
-    return [...super.styles(), base_css]
+  override stylesheets(): StyleSheetLike[] {
```

Comment:
@philippjfr, as I mentioned in the meeting, this is the change that affects panel.